pytato/distributed/nodes.py

```diff
@@ -106,7 +107,7 @@ def copy(self, **kwargs: Any) -> DistributedSend:
 
 # {{{ send ref holder
 
-@attrs.frozen(eq=False, repr=False, init=False, hash=True)
+@attrs.frozen(eq=False, repr=False, hash=True)
```

Comment:
```suggestion
@attrs.frozen(eq=False, repr=False, hash=True, cache_hash=True)
```
(This is the only `DistributedSendRefHolder` change that would be left in #512 after these changes, might as well do it here.)

Reply:
Done, thanks!